audio-worklet/src/noisegate-script-processor.js

```diff
@@ -16,18 +16,30 @@
 class NoiseGate {
   /**
    * A Noise gate allows audio signals to pass only when the registered volume
-   * is above a specified threshold.
+   * is above a specified threshold. This class is a wrapper around the script
+   * processor node.
+   * @class  NoiseGate
+   * @constructor
    * @param {BaseAudioContext} context the audio context
    * @param {Number} options.channels the number of input and output
-   *                                  channels, a maximum of two
+   *                                  channels. The default value is one, and
+   *                                  the implementation supports
+   *                                  a maximum of two channels.
    * @param {Object} options parameters for the noise gate
-   * @param {Number} options.attack seconds for gate to fully close
-   * @param {Number} options.release seconds for gate to fully open
-   * @param {Number} options.bufferSize the size of an onaudioprocess window
+   * @param {Number} options.attack seconds for gate to fully close. The default
+   *                                is zero.
+   * @param {Number} options.release seconds for gate to fully open. The default
+   *                                 is zero.
+   * @param {Number} options.bufferSize the size of an onaudioprocess window.
+   *                                    The default lets the script processor
+   *                                    decide.
    * @param {Number} options.timeConstant seconds for envelope follower's
-   *                                   smoothing filter alpha weight
+   *                                   smoothing filter alpha weight. The
+   *                                   default has been set experimentally to
+   *                                   0.0025s.
    * @param {Number} options.threshold decibel level beneath which sound is
-   *                                   muted
+   *                                   muted. The default is the maximum level
+   *                                   in dBFS, which is 0.
```

Comment:
Nit: Just say the default is 0 dBFS. No need to say anything about maximum level.